test/models/build_test.rb

```diff
@@ -229,8 +234,6 @@ def valid_build(attributes = {})
     end
 
     it 'cancels builds that have been running for too long' do
-      puts build.created_at
```

Comment:
😓

Reply:
thought that was mine, did not even blame it :D